### PR TITLE
Changes for SNAP-2604 & SNAP-3087:

### PR DESCRIPTION
**Pull request by snappy-sachin**
## What changes were proposed in this pull request?

 - JavaScript and CSS changes for displaying clusters start date and time and uptime on UI.

## How was this patch tested?

 - Tested manually

Please review http://spark.apache.org/contributing.html before opening a pull request.

### CODE REVIEW

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js [hunk 1]
```diff
@@ -4,6 +4,62 @@ var isAutoUpdateTurnedON = true;
...
+  var clusterStartTime = $("#hiddenData").data("clusterstarttime");
+  var start_date = new Date(clusterStartTime);
+  var now_date = new Date();
+
+  var seconds = Math.floor((now_date - start_date)/1000);
+  var minutes = Math.floor(seconds/60);
+  var hours = Math.floor(minutes/60);
+  var days = Math.floor(hours/24);
+
+  hours = hours-(days*24);
```
Comment (shirish-deshmukh) [inline core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js]: Formatting needs to be corrected. Space before/after - (minus) and *. Also at other places in this file

**Review by dshirish — COMMENTED** [review]
On my laptop UI layout does not look as intended (start date and uptime is hidden under "Total CPU Cores").

**Comment by snappy-sachin** [comment]
> On my laptop UI layout does not look as intended (start date and uptime is hidden under "Total CPU Cores").

clean build worked on your laptop..

File: core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js (hunk 1 above)
Comment (snappy-sachin) [reply core/src/main/resources/org/apache/spark/ui/static/snappydata/snappy-dashboard.js]: Done

**Review by dshirish — APPROVED** [review]
LGTM

**Comment by snappy-sachin** [comment]
Feedback taken care. Merging..

**Pull request merged**